README.md

```diff
@@ -20,7 +20,7 @@ The formal system which we are using as a target system is Lean's dependent type
 The challenge in the blog post is to formalise its `Theorem 1.1`, a variant of `Analytic 9.1` (i.e. Theorem 9.1 of [`Analytic.pdf`](https://www.math.uni-bonn.de/people/scholze/Analytic.pdf)) in Lean.
 We chose to use Lean 3 because of the advanced state of its classical mathematics library `mathlib`, an essential ingredient.
```

Comment:
This sentence is basically the same as the last one of the previous paragraph. Do we want the repetition?

Reply:
Good point.
```suggestion
```